CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ NOTE: [`epaint`](crates/epaint/CHANGELOG.md), [`eframe`](crates/eframe/CHANGELOG
 * Add `Plot::clamp_grid` to only show grid where there is data ([#2480](https://github.com/emilk/egui/pull/2480)).
 * Add `ScrollArea::drag_to_scroll` if you want to turn off that feature.
 * Add `Response::on_hover_and_drag_cursor`.
+* Add `Slider::snap_values`, which lets the user define certain points to snap the slider to when clicking or dragging ([#2523](https://github.com/emilk/egui/pull/2523)).
```

Comment:
This needs to be moved up to the `Unreleased` section.

Next time, open your PR from another branch then `master`, then I can help out by adding my own commits!

Anyways, after this I will merge it (finally!).

Thank you so much for your work an patience!